src/CodeGenerators/TimedScopeGen/TimedScopeClassTemplate.cs

```diff
@@ -5,7 +5,7 @@
 // <auto-generated>
```

Comment:
>auto-generated [](start = 4, length = 14)

This is an autogenerated file. You'll need to modify the template to ensure these changes persist